.pylintrc

```diff
@@ -1,38 +1,70 @@
-# Pylint 2.1.x - 2.2.x configuration file
+# Pylint 2.10.x configuration file
 #
 # This file is generated by l2tdevtools update-dependencies.py, any dependency
 # related changes should be made in dependencies.ini.
-[MAIN]
+[MASTER]
```

Comment:
is this changed to master because you generated the file in a branch and that is the default? shouldn't this be the MAIN branch here?

Reply:
This is from the .pylintrc file config, don't know why it was changed to MAIN before: https://www.codeac.io/documentation/pylint-configuration.html